src/ast.rs

```diff
@@ -32,10 +32,10 @@ struct Replacement {
 }
 
 /// Solidity AST representation.
-/// There are two fields, `element`
-/// which is the underlying json object representing
-/// an AST node and `contract` which indicates
-/// the name of the contract that this node belongs to.
+///
+/// There are two fields, `element` which is the underlying json object
+/// representing an AST node and `contract` which indicates the name of the
+/// contract that this node belongs to.
```

Comment:
Defaulting to 80-char lines, hope this is okay: it's blowing up diffs a bit, but I think it makes comments more readable